base/applications/notepad/lang/en-US.rc

```diff
@@ -167,6 +167,7 @@ task.\nClose one or more applications to increase the amount of\nfree memory."
     STRING_LINE_COLUMN "Line %d, column %d"
     STRING_PRINTERROR "Cannot print the file '%s'.\n\nMake sure that the printer is turned on and is configured properly."
     STRING_DEFAULTFONT "Lucida Console"
+    STRING_LINE_NUMBER_OUT_OF_RANGE "The specified line number is out of range."
```

Comment:
:+1: That's a better translation (and straight to the point) than Windows' one.

Reply:
Yes, we are making better Windows.